datasophon-worker/src/main/java/com/datasophon/worker/actor/InstallServiceActor.java

```diff
@@ -63,12 +63,10 @@ public void onReceive(Object msg) throws Throwable {
                 }
                 ExecResult execResult = ShellUtils.execWithStatus(Constants.INSTALL_PATH, commands, 180);
                 if (execResult.getExecResult()) {
-                    installResult = serviceHandler.install(command.getPackageName(), command.getDecompressPackageName(),
-                            command.getPackageMd5(), command.getRunAs());
+                    installResult = serviceHandler.install(command);
```

Comment:
use CacheUtils.getString(Constants.HOSTNAME) to get local hostname ,  use PropertyUtils.getString(Constants.MASTER_HOST)  to get master hostname